src/main/scala/com/microsoft/ml/spark/vw/VowpalWabbitContextualBandit.scala

```diff
@@ -275,7 +275,7 @@ class VowpalWabbitContextualBandit(override val uid: String)
     })
   }
 
-  override def fit(dataset: Dataset[_], paramMaps: Array[ParamMap]): Seq[VowpalWabbitContextualBanditModel] = {
```

Comment:
why is the override removed?

Reply:
Spark ML Lib changed Array[ParamMap] to Seq[ParamMap]
Update: keep "override", changed Array[] to Seq[]